cache-filter/src/unique_callout.rs

```diff
@@ -60,6 +60,7 @@ pub enum WaiterAction {
 }
 
 // This enum is used to give out status for an http context trying to get callout-lock.
+#[cfg(feature = "unique_callout")]
```

Comment:
Isn't this module already only compiled in when this is true?

Reply:
Forgot to remove it :weary: